docs/typescript/production-deploy.md

```diff
@@ -98,22 +98,16 @@ Send logs and errors to a logging service, so that when things go wrong, you can
 
 For more information about sending logs, see [Logging](/typescript/logging).
 
-## Metrics
+## Metrics and tracing
 
 ### Options
 
-Workers can emit metrics and traces. There are two [output options](https://github.com/temporalio/sdk-typescript/blob/9dd17554f3fa514f501d906da26cf710020bf34d/packages/core-bridge/index.d.ts#L74-L98) that can be provided to [`Runtime.install`](https://typescript.temporal.io/api/classes/worker.runtime/#install):
+Workers can emit metrics and traces. There are a few [temlemetry options](https://typescript.temporal.io/api/interfaces/worker.TelemetryOptions) that can be provided to [`Runtime.install`](https://typescript.temporal.io/api/classes/worker.runtime/#install), the common ones being:
 
-- `oTelCollectorUrl`: The URL of a gRPC [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
-- `prometheusMetricsBindAddress`: Address on the Worker host that will have metrics for [Prometheus](https://prometheus.io/) to scrape.
+- `metrics: { otel: { url } }`: The URL of a gRPC [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
+- `metrics: { prometheus: { bindAddress } }`: Address on the Worker host that will have metrics for [Prometheus](https://prometheus.io/) to scrape.
 
-There are three combinations of these options:
-
-- Only `oTelCollectorUrl` is specified: Metrics and traces are sent to the OpenTelemetry collector.
-- Both `oTelCollectorUrl` and `prometheusMetricsBindAddress` are specified: Traces are sent to the collector, and metrics are published for Prometheus.
-- Only `prometheusMetricsBindAddress` is specified: Only metrics are published for Prometheus.
```

Comment:
Is there an equivalent to these bullets?

Reply:
This isn't relevant anymore, see the change above

Reply:
When `metrics.otel` is supplied, does otel get metrics? When `metrics.otel` and `metrics.prometheus` are both supplied, does otel get metrics?

Reply:
It's supposed to be oneOf but apparently TS doesn't protect you from supplying both. I don't see a case for wanting both.
You can send traces to otel in addition to prometheus metrics but I didn't bother to mention this here because those traces wouldn't be relevant to 99.9% of users